plugins/jobs/server/job_rest.py

```diff
@@ -33,6 +33,8 @@ def __init__(self):
         self.route('GET', ('all',), self.listAllJobs)
         self.route('GET', (':id',), self.getJob)
         self.route('PUT', (':id',), self.updateJob)
+        self.route('GET', (':id', 'child',), self.getChildJobs)
```

Comment:
As discussed in person, a more consistent way of getting at "child resources" would be to add a `parentId` parameter to the `GET /job` endpoint.  This would follow the conventions of other model types.